### ISSUE TITLE
[WIP]Brainstorm Improvements for Mailer Contents

### ISSUE THREAD

**Issue by SidharthBansal** [publiclab/plots2#4109]
We have to improve the content of emails which users get on their email accounts.
*[ ] Add Descriptive Subjects
*[ ] Remove Redundancy
*[ ] Add an appropriate title
*[ ] Content enhancement
@publiclab/mentors what others we can include?
Thanks Lis Sylvan for your input in Open Hour.

**Comment by esylvan** [comment]
![temp](https://user-images.githubusercontent.com/3617884/49394485-e20abc00-f6e8-11e8-8dc9-105c501fbd02.PNG)

Some addition thoughts and suggestions:
- consider changing the From header to be something like "Public Lab" or "Public Lab notifications." 
- map out a design that provides clear subject and content sections. These could use brackets at the beginning of a subject line to indicate the type of the notification and the name/tag of it  
- think about how to make content more actionable with language at the beginning of the email body. For each type of notification, it might be a little different.
- as you add other ways to be notified (Twitter, LinkedIn, etc), the format should follow the same basic structure but be adapted to that particular medium.

**Comment by esylvan** [comment]
For your reference, t
![temp2](https://user-images.githubusercontent.com/3617884/49395124-80e3e800-f6ea-11e8-922e-5c4b5d6fd759.PNG)
his is what the body of a notification looks like in gmail.

**Comment by gauravano** [comment]
![Email_redesign](https://user-images.githubusercontent.com/20878070/49398771-5daa4e80-f765-11e8-8d41-47d174d1e243.gif)

**Comment by SidharthBansal** [comment]
@jywarren there are some other similar issues. Like 
![image](https://user-images.githubusercontent.com/20972099/50386545-d0786b00-070e-11e9-998e-a47c46ba4811.png) usernames are not linked with profile accounts.
This issue can be brainstormed and can be divided into multiple small issues. I don't think that it is an individual large project. It needs just brainstorming and some nice ftos to work upon.
I will be posting other issues here once I will find them.

**Comment by gauravano** [comment]
Let's start discussion with @esylvan's ideas:

> * consider changing the From header to be something like "Public Lab" or "Public Lab notifications."

We can use "PL notifications" too, or "Public Lab notifications" sounds good. 


> map out a design that provides clear subject and content sections. These could use brackets at the beginning of a subject line to indicate the type of the notification and the name/tag of it

 We should decide on the format of subject first and then subject line. What header we use also affects subject as we then have to choose subject accordingly.

> as you add other ways to be notified (Twitter, LinkedIn, etc), the format should follow the same basic structure but be adapted to that particular medium.

Public Lab blog's email has good footer for inspiration or a new footer can be designed.

@publiclab/community-reps ideas? 

Thanks!

**Comment by oorjitchowdhary** [comment]
This is great.. Even I find all the PL email notifications dull and boring. We can sort of make a template for all the outgoing emails and make the text descriptive and organized

**Comment by SidharthBansal** [comment]
You can give ideas and tell us what changes are required. If idea will be
liked then we can create a separate issue and will work on it.

**Comment by oorjitchowdhary** [comment]
Yes, I'm searching for some inspiration and will then select and suggest the best one suited for PL.

**Comment by oorjitchowdhary** [comment]
Here are some suggestions...

This one can be used for mentions maybe or replies/comments on a particular thread.
[image1.jpeg]

This one can be used for the email sent to a user subscribed to particular tag/tags
[image2.jpeg]

This one can be used for maybe the email to moderators.. (I'm also working on the active svg thing and hope to complete it soon!)
[image3.jpeg]

This one for posting a new research note/question/wiki
[image5.jpeg]

Hope this helps. Thanks!

**Comment by SidharthBansal** [comment]
Many emails from Public Lab are shown under Promotions. We need to do something here too. I missed to comment on the Dashboard proposal due to it. Sorry about that.
![image](https://user-images.githubusercontent.com/20972099/50648973-a478a780-0fa2-11e9-92b0-40c5a97d54d9.png)
If we see inside the mail then we can infer out redundant information at starting and end of mail
![image](https://user-images.githubusercontent.com/20972099/50649040-cbcf7480-0fa2-11e9-9d1f-e908337e91f4.png)

**Comment by jywarren** [comment]
Oh hmm, what causes things to be shown in promotions?

**Comment by SidharthBansal** [comment]
Google scans each mail. If it contains many links/images etc. then it is classified under Promotional email.
For complete note https://www.infusionsoft.com/business-success-blog/marketing/email-marketing/how-to-keep-your-emails-out-of-the-gmail-promotions-tab#

**Comment by SidharthBansal** [comment]
![image](https://user-images.githubusercontent.com/20972099/53739214-37f12a80-3eb7-11e9-9073-e7ab70c12007.png)
![image](https://user-images.githubusercontent.com/20972099/53739217-3d4e7500-3eb7-11e9-867c-5f2aa0164f7e.png)

There is no embedded links in the mail.

**Comment by SidharthBansal** [comment]
@jywarren can we create this as the hall of fame?
We may target some important features like emails from the same post should come below one another, etc.

**Comment by jywarren** [comment]
If we could improve the digest mailer template that'd be great. It has some
considerable formatting issues; tough but maybe not quite a hall of fame?
Do you get digests?

**Comment by SidharthBansal** [comment]
I have never worked with digests yet. Can make it hard GCI issue then?

**Comment by jywarren** [comment]
Sure!

**Comment by jywarren** [comment]
Maybe let's reshape this issue around the digest template? Two suggestions:

1. I saved a digest here. It doesn't look too great: https://gist.github.com/jywarren/08ab21cbdae2d45297f8c7de802c560f (below)
2. the digest page is supposed to use the same template so it's easier to debug this feature. But that didn't happen; it's just a list: https://publiclab.org/subscriptions/digest Let's make it use the same template, and then we can get them both looking great? 

We can reuse elements from the rest of the site even? Dunno... or the style guide? https://publiclab.org/notes/warren/05-07-2019/introducing-a-draft-style-guide-for-public-lab

![Screen Shot 2020-01-07 at 7 08 06 PM](https://user-images.githubusercontent.com/24359/71939584-2c2e5e80-3181-11ea-9916-d6b2dde74efa.png)

**Comment by SidharthBansal** [comment]
Published on GCI dashboard
